packages/py3_sphinx.rb

```diff
@@ -0,0 +1,21 @@
+require 'package'
+
+class Py3_sphinx < Package
```

Comment:
Does this conflict with the existing sphinx package?

Reply:
the existing sphinx package has been turned into this.